### ISSUE TITLE
Entering number in buffer after "[I" does not link to file

### ISSUE THREAD

**Issue by parkwherever** [moll/vim-node#21]
Instead, returns to file with cursor on initial word.

Is there a prefix needed/am I missing something here?

**Comment by moll** [comment]
Hey,

Got your mail, but I'll reply here so others in the future could see this. So, Node.vim doesn't actually implement that functionality itself. It only sets some required settings so Vim could find included files.

`[I` is a little weird indeed. I haven't used it often, so I had to look up `:h [I`. Not the most convenient interface. It doesn't actually allow you to give a count with `[I`. Instead it seems you got to do `{count} [ Ctrl-I]`. Like `1 [ Ctrl-I`.

Does that help?

PS. I just saw a possible bug where Vim couldn't continue searching deeper than one level of includes ("requires" in the CommonJS land). I'll see about that.
